cmd/utils/flags.go

```diff
@@ -1071,6 +1071,13 @@ var (
 		Usage:    "List of mock bls public keys which are reflect 1:1 with mock.validators",
 		Category: flags.MockCategory,
 	}
+
+	ConcurrentUpdateThresholdFlag = &cli.IntFlag{
+		Name:     "concurrencyupdatethreashold",
```

Comment:
nit: concurrent-update-threshold
I think this flag name is too long and a little bit hard to read so adding some dashes looks better